src/authService.js

```diff
@@ -20,6 +20,15 @@ export class AuthService {
     return this.rest.find(this.auth.getProfileUrl());
   }
 
+  setMe(profile) {
+    return this.rest.update(this.auth.getProfileUrl(), profile)
+      .then(response => {
+        return response;
+      }).catch(err => {
```

Comment:
This catch should be removed, it's up to the caller to handle exceptions. (I realize it's also in login, would you mind taking that one out, too?)